cove_360/lib/threesixtygiving.py

```diff
@@ -222,8 +224,11 @@ def produce_message(self):
 
     def get_heading_count(self):
         if self.grants_count == 1 and self.count == 1:
+            self.grants_percentage = int(100)
             return '1'
-        return '{:.0%} of'.format(self.count / self.grants_count)
+        heading_percentage = '{:.0%} of'.format(self.count / self.grants_count)
+        self.grants_percentage = int(heading_percentage[:-1])
```

Comment:
This doesn't work, because `heading_percentage` contains the string of, so `heading_percentage[:-1]` is e.g. `14% o`.

Reply:
Right! I have fixed that.